### PR TITLE
Add slider "sliding" sample support

### LINKED ISSUE #8790: Slider slide legacy hitsound support
**Describe the new feature:** 
When trying out a couple maps which utilise the sliderslide legacy hitsound in standard.
(e.g. [Fort's Ooi](https://osu.ppy.sh/beatmapsets/355573#osu/784791), and a [taikosu! map](https://osu.ppy.sh/beatmapsets/25836#osu/87484). The slider slide aren't utilised in the body of the sliders.

**Proposal designs of the feature:** 
Allowing `soft/normal/drum-sliderslide` from legacy hitsounds into slider bodies. This of course would work the same way as in stable.

### PR DESCRIPTION
**Pull request by peppy**
- Closes #8790
- [x] Depends on https://github.com/ppy/osu/pull/9645

Please test on https://osu.ppy.sh/beatmapsets/861#osu/5680

### CODE REVIEW

**Comment by bdach** [comment]
From looking at the map linked `sliderslide` support seems to be working but that map also uses `sliderwhistle` which is not supported, so no full parity yet it seems. Wiki also tells me `slidertick` exists.

**Comment by peppy** [comment]
Hmmm well spotted..... adding the rest is probably best for a follow-up PR. I need to encapsulate this logic better, I think (sliderslide / sliderwhistle / spinnerspin are all going to be doing very similar things, so may be better to move to a dedicated drawable component)

Do we not already support slidertick, though?

**Comment by bdach** [comment]
Good point. Admittedly I haven't checked `slidertick`, just brought it up as it was in the same section as the others on the wiki. It does look to have support.

File: osu.Game.Rulesets.Osu/Objects/Drawables/DrawableSlider.cs [hunk 1]
```diff
@@ -81,6 +83,41 @@ private void load()
...
+            Tracking.BindValueChanged(updateSlidingSample);
+        }
+
+        private SkinnableSound slidingSample;
+
+        protected override void LoadSamples()
+        {
+            base.LoadSamples();
+
+            slidingSample?.Expire();
```
Comment (smoogipoo) [inline osu.Game.Rulesets.Osu/Objects/Drawables/DrawableSlider.cs:96]: Is this a performance optimisation to stop updates?

Comment (peppy) [reply osu.Game.Rulesets.Osu/Objects/Drawables/DrawableSlider.cs:96]: Check where this is called (generally on sample change, editor for now). A new instance is being constructed just below. We need to kill the last one.

Comment (smoogipoo) [reply osu.Game.Rulesets.Osu/Objects/Drawables/DrawableSlider.cs:96]: Aaaaaah I misread the method as `LoadComplete`

**Review by smoogipoo — COMMENTED** [review]
I think there's more than I can comment on here like:
- Sample banks seem to only be used if beatmap version >= 14, which means this code should probably exist inside the beatmap converter.
- There's a lot of funny business around how the control point affects the sample in stable (I'm not actually 100% sure if it does in this case).

Both of these, I think, are issues that can be touched upon later if they're deemed necessary. Always having the sample here seems like a step forward.

**Review by smoogipoo — APPROVED** [review]

**Pull request merged**